src/domain/eventSearch/filterSummary/FilterSummary.tsx

```diff
@@ -74,6 +75,10 @@ const FilterSummary: React.FC<Props> = ({ onClear, route }) => {
         type === 'category'
           ? categories.filter((category) => category !== value)
           : categories,
+      hobbyTypes:
+        type === 'hobbyType'
+          ? hobbyTypes?.filter((hobbyType) => hobbyType !== value)
+          : hobbyTypes,
```

Comment:
Noticed some repetition here with filtering, you didn't do these but now is a good change to add helper function to avoid repetition.. for example something like this (can be maybe improved still):

```
    const getFilteredList = (list: string[], listType: FilterType) =>
      type === listType ? list.filter((v) => v !== value) : list;

    const search = getSearchQuery({
      categories: getFilteredList(categories, 'category'),
```

Reply:
i ll check